scribbler/templatetags/scribbler_tags.py

```diff
@@ -83,7 +83,9 @@ def render(self, context):
         context['can_edit_scribble'] = can_edit
         context['can_delete_scribble'] = can_delete
         context['raw_content'] = self.raw
-        return wrapper_template.render(context)
+        # render() takes a dict, so we have to extract the context dict from the object
+        context_data = context.dicts[-1]
+        return wrapper_template.render(context_data)
```

Comment:
This is probably what's causing the CSRF error, and again on line 201.  I can extract that information (Along with the request, permissions, etc) from the context object and pass it to render's request parameter and hopefully that'll clear up the CSRF issue.